test/e2e/syncer/syncer_test.go

```diff
@@ -251,6 +255,76 @@ func TestSyncerLifecycle(t *testing.T) {
 			return true
 		}, wait.ForeverTestTimeout, time.Millisecond*100, "upstream configmap %s/%s was not found", upstreamNamespace.Name, expectedConfigMapName)
 
+		// Tunnels KCP <----> Syncer
+		t.Logf("Create upstream service account permissions for downstream in-cluster tunnel pods access")
+		podsAllRole := &rbacv1.Role{
+			ObjectMeta: metav1.ObjectMeta{Name: "pods-all"},
+			Rules: []rbacv1.PolicyRule{
+				rbachelper.NewRule("*").Groups("").Resources("pods").RuleOrDie(),
+				rbachelper.NewRule("*").Groups("").Resources("pods/log").RuleOrDie(),
+				rbachelper.NewRule("*").Groups("").Resources("pods/exec").RuleOrDie(),
+			},
+		}
+
+		_, err = upstreamKubeClusterClient.RbacV1().Roles(upstreamNamespace.Name).Create(logicalcluster.WithCluster(ctx, wsClusterName), podsAllRole, metav1.CreateOptions{})
```

Comment:
this confuses me. Why do we need upstream rbac?

Reply:
Would have expected them downstream for the syncer service account only.

Reply:
I was thinking they were synced, and is how an user will use them ... are they not synced?

Reply:
oh, the service accounts are synced, not the roles, damn

Reply:
yes